examples/relationship_example.py

```diff
@@ -1,3 +1,7 @@
+"""
+# Examples for relationships in FIWARE ContextBroker
```

Comment:
maybe it would good to add a little more description. this exmaple is actually stolen form the FIWARE step-by step